cg/apps/lims/api.py

```diff
@@ -9,8 +9,14 @@
 from genologics.lims import Lims
 from requests.exceptions import HTTPError
 
-from cg.constants import Priority
-from cg.constants.lims import MASTER_STEPS_UDFS, PROP2UDF, DocumentationMethod, LimsArtifactTypes
+from cg.constants.constants import ControlOptions, CustomerId
```

Comment:
I get an error becaure `Priority` got removed from the imports. It should be added again